learningresources/api.py

```diff
@@ -128,7 +128,10 @@ def get_repos(user_id):
     Returns:
         repos (query set of learningresource.Repository): Repositories
     """
-    user = User.objects.get(id=user_id)
+    try:
+        user = User.objects.get(id=user_id)
+    except User.DoesNotExist:
+        return []
```

Comment:
Since we have `PermissionDenied` declared above, it would be better to raise it here. If a developer has a bug and is passing a bad value here, it'll silently "fail" but return a value that _looks like_ what they expect. That could cause them to waste a lot of time figuring out what went wrong (or not notice at all and ship the code to production).